src/css/colors.scss

```diff
@@ -47,6 +47,7 @@ $orange-alpha: #fca714;
   --blue-800: #202750;
 
   // GRAY
+  --gray-border: #DFE4ED;
```

Comment:
I am not too sure about the naming for this color variable.
I think we should stick to previously defined color naming conventions like `--gray-90` or `--gray-102` whatever group it fits.
Color variable names are hard to get, worse than any other variable.
I think we should treat these variables the following way
 `--[primary_color_group_name]-[subgroup_index][color_index]`
so an example `--gray-102` is
- primary_color_group_name: **gray**
- subgroup_index: **1**
- color_index: **02**

The issue I have with `--gray-border` is that we might want to use this color `#DFE4ED` for something else than a border also what if in the future we need to create a variable for a different border color like `#636D82` do we call it
 `--gray-border-2`?